llpc/tool/amdllpc.cpp

```diff
@@ -1765,6 +1767,148 @@ void FindAllMatchFiles(
     return;
 }
 #endif
+
+// =====================================================================================================================
+// Builds necessary variants of the shader inFile.  It must be either spir-v assembly or spir-v binary.
+static Result BuildShaderVariants(
+    ICompiler*            pCompiler,   // [in] LLPC context
+    const std::string& inFile)        // Input filename
+{
+    Result result = Result::Success;
+    CompileInfo compileInfo = {};
+    std::string fileNames;
+    compileInfo.doAutoLayout = true;
```

Comment:
I suppose do auto-layout is a temp solution for shader variants, if yes, could you help to add a comment for it?